guides/_subsections/using-the-cryostat-agent.md

```diff
@@ -16,4 +16,4 @@ The programming interfaces for Cryostat and its Agent are designed to implement
     <li>Once the Agent has been installed or attached to the running JVM instance, it can begin collecting data and sending it to Cryostat for analysis.</li>
 </ol>
 
-[comment]: # TODO continue explaining how to include the Agent in application builds, how to configure it
+For instructions on how to install the Cryostat Agent into your applications, [check the Getting Started section](/get-started/#using-the-cryostat-agent).
```

Comment:
> [check the Getting Started section](/get-started/#using-the-cryostat-agent).

Getting Started is a whole page by itself right? Maybe something like `Setup section in Getting Started`?